src/main/scala/tilelink/WidthWidget.scala

```diff
@@ -28,27 +29,27 @@ class TLWidthWidget(innerBeatBytes: Int)(implicit p: Parameters) extends LazyMod
       val hasData = edgeIn.hasData(in.bits)
       val limit   = UIntToOH1(size, keepBits) >> dropBits
 
-      val count  = RegInit(UInt(0, width = countBits))
-      val first  = count === UInt(0)
+      val count  = RegInit(0.U(countBits.W))
+      val first  = count === 0.U
       val last   = count === limit || !hasData
-      val enable = Seq.tabulate(ratio) { i => !((count ^ UInt(i)) & limit).orR }
+      val enable = Seq.tabulate(ratio) { i => !((count ^ i.U) & limit).orR }
 
-      val corrupt_reg = RegInit(Bool(false))
+      val corrupt_reg = RegInit(false.B)
       val corrupt_in = edgeIn.corrupt(in.bits)
       val corrupt_out = corrupt_in || corrupt_reg
 
       when (in.fire()) {
-        count := count + UInt(1)
+        count := count + 1.U
         corrupt_reg := corrupt_out
         when (last) {
-          count := UInt(0)
-          corrupt_reg := Bool(false)
+          count := 0.U
+          corrupt_reg := false.B
         }
       }
 
       def helper(idata: UInt): UInt = {
-        val odata = Seq.fill(ratio) { idata }
-        val rdata = Reg(Vec(ratio-1, idata))
+        val odata = Seq.fill(ratio) { WireInit(idata) }
```

Comment:
@aswaterman , @hcook this is another place where I would like a sanity check on my translation... is this equivalent?

Reply:
I don't think this one should be necessary, given that odata is never assigned to.  If you back out this particular change, what goes wrong?

Reply:
it complains that `idata` needs to be hardware. Let me back it out and double check

Reply:
Yeah, I see it.  But I think this is actually masking a different bug.  I'm looking into it.

Reply:
sorry fighting merge conflicts will make this PR against master as well, performing the LEC checks there. Haven't gotten to try backing this out yet